cmd/kubeadm/app/preflight/checks_test.go

```diff
@@ -248,7 +248,8 @@ func TestRunInitNodeChecks(t *testing.T) {
 		actual := RunInitNodeChecks(exec.New(), rt.cfg, sets.NewString(), rt.isSecondaryControlPlane, rt.downloadCerts)
 		if (actual == nil) != rt.expected {
 			t.Errorf(
-				"failed RunInitNodeChecks:\n\texpected: %t\n\t  actual: %t\n\t error: %v",
+				"failed RunInitNodeChecks:%v\n\texpected: %t\n\t  actual: %t\n\t error: %v",
+				rt.name,
```

Comment:
ok, i had a look at the level of refactor this requires in it's not that trivial.
we have code paths that are still tested here, so i'm going to log a ticket in k/kubeadm with a brief plan on how to proceed.

in the meantime we can merge this PR, but please add a similar "rt.name" in TestRunJoinNodeChecks.

Reply:
Yes! Thanks so much, I was just about to fix that :-D  I'll fix the case with no name.

Reply:
@RobertKielty I think the ask here is to update https://github.com/kubernetes/kubernetes/pull/93885/files#diff-a54a9514dfa4dd1ab4cc9ada7445c91dR297 below with `rt.name` as well :slightly_smiling_face:

Reply:
yes, please add  "rt.name" in TestRunJoinNodeChecks

Reply:
I misread this feedback, will add the rt.name TestRunJoinNodeChecks now.